src/components/BoardHeader/HeaderMenu/BoardSettings/BoardSettings.scss

```diff
@@ -50,6 +51,7 @@ $header-menu-button-width: 24px;
   padding: 0;
   transition: border-color 0.1s linear;
   caret-color: $color-white;
+  min-width: 0;
```

Comment:
Omitting this doesn't seem to change anything, even with an empty board name.
Is it required?

Reply:
For me it's only Firefox that's making problems here